docs/adr/adr-007-data-retention.md

```diff
@@ -0,0 +1,196 @@
+# Decision Record: Data Retention
+
+## Context
+
+The original ADR concerning data retention resides [here][initial policy] and
+outlines our approach from February 2018. At that point in time it was decided
+we should retain all data produced by EmailAlertApi with the exception of
```

Comment:
```suggestion
we should retain all data in EmailAlertApi with the exception of
```

Minor: we still retained the data, but it was in a different place.

Reply:
EmailAlertApi should be Email Alert API.